profiles/package.mask

```diff
@@ -33,6 +33,137 @@
 
 #--- END OF EXAMPLES ---
 
+# hololeap <hololeap@protonmail.com> (2022-08-12)
+# Monolithic mask for dev-haskell/* packages which have no reverse dependencies,
+# are broken, or severely out of date. The aim is to have the Haskell overlay
+# (::haskell) be the place for development packages and only have packages
+# needed for end-user applications in ::gentoo, as the status quo has
+# proven to be unsustainable. More up-to-date versions of these packages
+# are available in ::haskell.
+# Removal on 2022-09-12.
+app-portage/fquery
+dev-games/goatee
+dev-haskell/megaparsec
+dev-haskell/parser-combinators
+dev-haskell/megaparsec-tests
+dev-haskell/hspec-megaparsec
+dev-haskell/neat-interpolation
+dev-haskell/stack
+dev-haskell/abstract-deque-tests
+dev-haskell/aeson-diff
+dev-haskell/annotated-wl-pprint
+dev-haskell/authenticate-oauth
+dev-haskell/base-prelude
+dev-haskell/base-unicode-symbols
+dev-haskell/cabal-install
+dev-haskell/casa-client
+dev-haskell/casa-types
+dev-haskell/cipher-aes128
+dev-haskell/concurrent-extra
+dev-haskell/configurator
+dev-haskell/contravariant-extras
+dev-haskell/convertible
+dev-haskell/cpphs
+dev-haskell/crypto-pubkey-types
+dev-haskell/cryptohash
+dev-haskell/cryptohash-cryptoapi
+dev-haskell/data-default-instances-base
+dev-haskell/dns
+dev-haskell/drbg
+dev-haskell/edit-distance-vector
+dev-haskell/either
+dev-haskell/errorcall-eq-instance
+dev-haskell/filelock
+dev-haskell/filemanip
+dev-haskell/filepattern
+dev-haskell/fsnotify
+dev-haskell/ghc-lib-parser
+dev-haskell/ghc-lib-parser-ex
+dev-haskell/githash
+dev-haskell/groupoids
+dev-haskell/groups
+dev-haskell/hackage-security
+dev-haskell/haddock
```

Comment:
Surprisingly, the `haddock` executable is not needed for building documentation for Haskell packages. The built-in `haddock` command of Cabal is used internally instead.